src/backend/src/utils/validation.utils.ts

```diff
@@ -44,3 +45,9 @@ export const isWorkPackageStageOrNone = (validationObject: ValidationChain): Val
 export const isAccount = (validationObject: ValidationChain): ValidationChain => {
   return validationObject.isString().isIn([ClubAccount.BUDGET, ClubAccount.CASH]);
 };
+
+export const isMaterialStatus = (validationObject: ValidationChain): ValidationChain => {
+  return validationObject
+    .isString()
+    .isIn([Material_Status.ORDERED, Material_Status.RECEIVED, Material_Status.SHIPPED, Material_Status.UNORDERED]);
```

Comment:
you should create a MaterialStatus shared type and use that here instead of the prisma type